lib/dpl/provider/pypi.rb

```diff
@@ -32,6 +32,12 @@ def skip_upload_docs?
           (options.has_key?(:skip_upload_docs) && options[:skip_upload_docs])
       end
 
+      def pypi_skip_existing_option
+          if options.has_key?(:skip_existing) && options[:skip_existing]
```

Comment:
It would look nicer in form of `options.fetch(:skip_existing, false)`

Reply:
Done, all jobs mentioned have been rebuilt with the same outcome.